packages/gatsby-theme-newrelic/gatsby/on-pre-render-html.js

```diff
@@ -3,22 +3,37 @@ import path from 'path';
 import { withPrefix } from 'gatsby';
 import { getTessenConfig } from '../src/utils/config';
 import { getTessenPath } from './constants';
-import OneTrust from '../src/components/OneTrust';
 
 const onPreRenderHTML = (
-  { getHeadComponents, replaceHeadComponents },
+  { getHeadComponents, replaceHeadComponents, pathname },
   themeOptions
 ) => {
   const tessen = getTessenConfig(themeOptions);
-  const { oneTrustID } = themeOptions;
+  const languages = { kr: 'ko', jp: 'ja', en: 'en' };
```

Comment:
`nit` we can probably get rid of 'en' since it'll never get used, right?

Reply:
ahh yes good point